bin/render-activity-stream-html.js

```diff
@@ -108,6 +110,7 @@ for (const src of ${JSON.stringify(scripts, null, 2)}) {
     <div id="snippets-container">
       <div id="snippets"></div>
     </div>${options.noscripts ? "" : scriptTag}
+    ${isPrerendered ? "" : isPrerenderedStatus}
```

Comment:
@Mardak Agree reusing check in root tag is much better, my initial attempt to reuse that space didn’t work probably due to some other issue because of which ended up putting comment in its own check. 

I have updated code https://github.com/mozilla/activity-stream/pull/4691 and verified the comment shows for discovery stream when we are not using pre-rendered version. Thanks